src/lib/rest/rest.cpp

```diff
@@ -1778,6 +1780,7 @@ void restInit
   mhdConnectionTimeout = _mhdTimeoutInSeconds;
 
   strncpy(restAllowedOrigin, _allowedOrigin, sizeof(restAllowedOrigin));
+  strncpy(restCORSEnabled, _corsEnabled, sizeof(restCORSEnabled));
```

Comment:
If you indent the `_corsEnabled` to be under `_allowedOrigin`, it gets a little easier on the eye. Same for the third parameter.

Reply:
Fixed in de7d90b